docs/modules/Either.ts.md

```diff
@@ -370,22 +440,51 @@ Added in v2.6.3
 
 ## sequence
 
+Evaluate each monadic action in the structure from left to right, and collect the results.
+
 **Signature**
 
 ```ts
 export declare const sequence: Sequence2<'Either'>
 ```
 
+**Example**
+
+```ts
+import { pipe } from 'fp-ts/function'
+import * as E from 'fp-ts/Either'
+import * as O from 'fp-ts/Option'
+
+assert.deepStrictEqual(pipe(E.right(O.some('a')), E.sequence(O.option)), O.some(E.right('a')))
+
+assert.deepStrictEqual(pipe(E.right(O.none), E.sequence(O.option)), O.none)
```

Comment:
this baffles me as noob... if I can have a right of none then why can I not have a some of right of none?

also would be great if there was a left example....